config/sync/role_watchdog.settings.yml

```diff
@@ -0,0 +1,6 @@
+_core:
+  default_config_hash: bMLk1BMYZdizZNW-Bn-_bY0fSNgbC7yjhZfQ4zpT0aA
+role_watchdog_monitor_roles:
+  administrator: administrator
+role_watchdog_notify_email: email@example.com
```

Comment:
Do we have a monitored email to use here?  Can we send email?

Reply:
No, we have no ability send outgoing email. These are being logged to watchdog which (eventually) will be sent to cloud.gov logs.